mmv1/templates/terraform/post_create/compute_backend_bucket_security_policy.go.erb

```diff
@@ -0,0 +1,19 @@
+// security_policy isn't set by Create / Update
+if o, n := d.GetChange("edge_security_policy"); o.(string) != n.(string) {
+		pol, err := ParseSecurityPolicyFieldValue(n.(string), d, config)
+  if err != nil {
+    return errwrap.Wrapf("Error parsing Backend Service security policy: {{err}}", err)
+  }
+
+  spr := emptySecurityPolicyReference()
+  spr.SecurityPolicy = pol.RelativeLink()
+  op, err := config.NewComputeClient(userAgent).BackendBuckets.SetEdgeSecurityPolicy(project, obj["name"].(string), spr).Do()
```

Comment:
Can we do this through the HTTP client directly? It feels a little awkward to add a dependency on the client libraries to a mmv1-based resource

Reply:
This is already a pattern in another resource https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/templates/terraform/post_create/compute_backend_service_security_policy.go.erb#L8-L9

There is where I adapted the code from